app/javascript/src/ChangePassword/components/ChangePassword.jsx

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React, { useState } from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import { LoginPage, Form, ActionGroup, Button } from '@patternfly/react-core'
+import { HiddenInputs, FlashMessages } from 'LoginPage'
+import { PasswordInput } from 'ChangePassword'
+import type { FlashMessage } from 'Types'
+
+import 'LoginPage/assets/styles/loginPage.scss'
+import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
+import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
+
+type Props = {
+  lostPasswordToken: ?string,
+  url: string,
+  errors: (?FlashMessage)[]
+}
+
+const ChangePassword = ({lostPasswordToken, url, errors}: Props) => {
+  const [password, setPassword] = useState('')
+  const [isPasswordValid, setIsPasswordValid] = useState(undefined)
+  const [passwordConfirmation, setPasswordConfirmation] = useState('')
+  const [isPasswordConfirmationValid, setIsPasswordConfirmationValid] = useState(undefined)
+  const formDisabled = !isPasswordValid || !isPasswordConfirmationValid
+
+  return (
+    <LoginPage
+      brandImgSrc={brandImg}
+      brandImgAlt='Red Hat 3scale API Management'
+      backgroundImgSrc={PF4DownstreamBG}
+      backgroundImgAlt='Red Hat 3scale API Management'
+      loginTitle='Change Password'
+      footer={null}
+    >
+      {errors && <FlashMessages flashMessages={errors}/>}
+      <Form
+        action={url}
+        noValidate
+        id='edit_user_2'
+        acceptCharset='UTF-8'
+        method='post'
+      >
+        <input type='hidden' name='_method' value='put' />
+        <HiddenInputs />
+        <PasswordInput isRequired name='password' label='Password'
```

Comment:
Consistency in code is important for a good readability and I will never quit fighting for this 😄. Let's put properties 1 per line and never starting in the first line, just like Form a couple of lines above.
When properties are short enough, they can be put in one line.